discovery_engine_core/web-api/src/bin/ingestion.rs

```diff
@@ -76,6 +77,31 @@ struct IngestedDocument {
     properties: DocumentProperties,
 }
 
+#[derive(Debug, Clone, Serialize)]
+struct IngestionError {
+    /// List of Document Indices which were not successfully processed
+    documents: Vec<ErroredDocumentId>,
+}
+
+#[derive(Serialize, Debug, Clone)]
+pub struct ErroredDocumentId {
+    id: DocumentId,
+}
+
+impl IngestionError {
+    pub(crate) fn new(errored_ids: Vec<DocumentId>) -> Self {
+        Self {
+            documents: errored_ids
+                .into_iter()
+                .map(|id| ErroredDocumentId { id })
+                .collect_vec(),
+        }
+    }
+    pub(crate) fn to_reply(&self) -> impl Reply {
```

Comment:
one line of space would be nice
```suggestion

    pub(crate) fn to_reply(&self) -> impl Reply {
```